webpack-modules/README.md

```diff
@@ -0,0 +1,31 @@
+# WebExtension Webpack Example
+A minimal example of how to use npm modules from within a WebExtension.
+The example package used by this extension is `left-pad`, an essential package
+in almost any situation.
+
+## What it does
+This setup takes a background and a content script entrypoint and outputs them
```

Comment:
I think it would be good to split "what it does" in two parts here:
* what this example does: shows how to package npm modules for a WebExtension
* what the WebExtension does